scavenger-agent-java/src/integrationTest/java/integrationTest/javaagent/ConfigTest.java

```diff
@@ -0,0 +1,67 @@
+package integrationTest.javaagent;
+
+import static integrationTest.util.AgentLogAssertionUtil.assertDisabled;
+import static integrationTest.util.AgentLogAssertionUtil.assertSampleAppOutput;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+
+import java.util.Properties;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import integrationTest.support.AgentIntegrationTestContextProvider;
+import integrationTest.support.AgentRunner;
+
+@ExtendWith(AgentIntegrationTestContextProvider.class)
+@DisplayName("config test")
+public class ConfigTest {
+
+    @TestTemplate
```

Comment:
interesting.. what is it for?

Reply:
It acts as a test template and invoked with contexts that `AgentIntegrationTestContextProvider` provides, which includes paths to various Java versions.